docs/standards.md

```diff
@@ -120,24 +120,26 @@ __All standard video.js plugins must implement the core set of npm scripts.__
 
 All names are lower-case and use colons (`:`) as sub-task separators (multiple colons separate multiple levels of sub-tasks). Other scripts (e.g., sub-sub-tasks and `pre*`/`post*` scripts) will be created as well, but these are not documented here as they are not considered core scripts and are subject to change.
 
-npm Script   | Optional | Description
------------- | -------- | -----------
-`build`      |          | Runs all build sub-tasks.
-`build:css`  | ✓        | Builds the Sass entry point.
-`build:js`   |          | Builds the Browserify entry point.
-`build:lang` | ✓        | Builds language files.
-`build:test` |          | Builds the test Browserify entry point.
-`clean`      |          | Cleans up _all_ build artifacts.
-`docs`       | ✓        | Performs documentation tasks.
-`lint`       |          | Lints all `.js` ES6 source file(s) using `videojs-standard`.
-`start`      |          | Starts a development server at port `9999` (or closest open port) and runs `watch`.
-`test`       |          | Runs `lint`, builds tests, and runs tests in available browsers.
-`test:*`     | ✓        | Browser-specific tests (e.g. `test:firefox`).
-`watch`      |          | Watches everything and runs appropriate tasks.
-`watch:css`  | ✓        | Triggers a build when the Sass entry point changes (without banner comment).
-`watch:js`   |          | Triggers a build when the Browserify entry point changes (without banner comment or minification).
-`watch:test` |          | Triggers a build when the test entry point changes.
-`version`    | ✓        | Special handling if Bower support is enabled.
+npm Script    | Optional | Description
+------------- | -------- | -----------
+`build`       |          | Runs all build sub-tasks.
+`build:css`   | ✓        | Builds the Sass entry point.
+`build:js`    |          | Builds the Browserify entry point.
+`build:lang`  | ✓        | Builds language files.
+`build:test`  |          | Builds the test Browserify entry point.
+`change`      | ✓        | Adds a free-form text entry to the CHANGELOG.
```

Comment:
Ah, cool. It should be documented that that is what happens. It isn't obvious to me.